html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">Email</label> <br />
+      <input type="text" name="email" size="15" maxlength="50"> <br />
+
+    <label for="coding">Why is coding cool?</label> <br />
+      <select name="coding">
+        <option value="select">Select One</option>
+        <option value ="optionone">Because its intersting</option>
+        <option value="optiontwo">Because its fun</option>
+        <option value="optionthree">Because its powerful</option>
+      </select> <br />
+
+    <label for="language">Favorite Language (choose one)<br />
+      <input type="radio" name="language" value="html" />
+        HTML
+      <input type="radio" name="language" value="css" />
+        CSS
+      <input type="radio" name="language" value="javascript" />
+        JavaScript
+    </label> <br />
+
+    <label for="info">Would you like to sign up for more information?</label> <br />
+      <input type="checkbox" name="yes" value="sign" /> Sign me up! <br />
```

Comment:
This `input` will need an `id` to match the `for` attribute on its `label`

Reply:
Whoops! The label here should be around "Sign me up!" since that is the text the user would click to put focus on the input